From 01f16203f768301e5a4647a5534e80db14da74b6 Mon Sep 17 00:00:00 2001
From: Satoru Takeuchi <satoru.takeuchi@gmail.com>
Date: Tue, 25 Sep 2018 03:21:17 +0900
Subject: [PATCH 01/10] 1/10 fine

---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index 5e13c544bbf4..ac2a58b92f96 100644
--- a/init/main.c
+++ b/init/main.c
@@ -590,6 +590,7 @@ asmlinkage __visible void __init start_kernel(void)
 	mm_init();
 
 	ftrace_init();
+	pr_warn("1/10");
 
 	/* trace_printk can be enabled here */
 	early_trace_init();
-- 
2.17.1

