From 72e36d3fbbe677aadb96686e095b1142f8398ae7 Mon Sep 17 00:00:00 2001
From: Satoru Takeuchi <satoru.takeuchi@gmail.com>
Date: Tue, 25 Sep 2018 03:22:38 +0900
Subject: [PATCH 03/10] 3/10 fine

---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index b9d0288e530d..09d16fd09215 100644
--- a/init/main.c
+++ b/init/main.c
@@ -592,6 +592,7 @@ asmlinkage __visible void __init start_kernel(void)
 	ftrace_init();
 	pr_warn("1/10");
 	pr_warn("2/10");
+	pr_warn("3/10");
 
 	/* trace_printk can be enabled here */
 	early_trace_init();
-- 
2.17.1

