From 9f6c5fbcd3276216291f60f41504bab6003c95e6 Mon Sep 17 00:00:00 2001
From: Satoru Takeuchi <satoru.takeuchi@gmail.com>
Date: Sat, 4 Feb 2017 21:01:30 +0900
Subject: [PATCH 07/10] 7/10: BUG

---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index 6ef4bfe0c76d..6f4893cfb8d6 100644
--- a/init/main.c
+++ b/init/main.c
@@ -664,6 +664,7 @@ asmlinkage __visible void __init start_kernel(void)
 	pr_warn("5/10 patch is applied!\n");
 	pr_warn("6/10 patch is applied!\n");
 	BUG();
+	pr_warn("7/10 patch is applied!\n");
 
 	/* Do the rest non-__init'ed, we're now alive */
 	rest_init();
-- 
2.11.0

