From ddd7cdeacf472a74d9143124b305cc515b212dda Mon Sep 17 00:00:00 2001
From: Satoru Takeuchi <satoru.takeuchi@gmail.com>
Date: Sat, 4 Feb 2017 21:01:47 +0900
Subject: [PATCH 08/10] 8/10: BUG

---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index 6f4893cfb8d6..f46f34302e56 100644
--- a/init/main.c
+++ b/init/main.c
@@ -665,6 +665,7 @@ asmlinkage __visible void __init start_kernel(void)
 	pr_warn("6/10 patch is applied!\n");
 	BUG();
 	pr_warn("7/10 patch is applied!\n");
+	pr_warn("8/10 patch is applied!\n");
 
 	/* Do the rest non-__init'ed, we're now alive */
 	rest_init();
-- 
2.11.0

