From a57fc8b827e195050aa96f143ebebd66ceb23b81 Mon Sep 17 00:00:00 2001
From: Satoru Takeuchi <satoru.takeuchi@gmail.com>
Date: Tue, 25 Sep 2018 03:48:48 +0900
Subject: [PATCH 08/10] 8/10 BUG

---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index 349157e9ed26..1fa980210d9c 100644
--- a/init/main.c
+++ b/init/main.c
@@ -598,6 +598,7 @@ asmlinkage __visible void __init start_kernel(void)
 	pr_warn("6/10");
 	BUG();
 	pr_warn("7/10");
+	pr_warn("8/10");
 
 	/* trace_printk can be enabled here */
 	early_trace_init();
-- 
2.17.1

