From 575c678ee76da1ba539bc73922653adffe9d85ee Mon Sep 17 00:00:00 2001
From: Satoru Takeuchi <satoru.takeuchi@gmail.com>
Date: Tue, 25 Sep 2018 03:49:02 +0900
Subject: [PATCH 09/10] 9/10 BUG

---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index 1fa980210d9c..53011afab325 100644
--- a/init/main.c
+++ b/init/main.c
@@ -599,6 +599,7 @@ asmlinkage __visible void __init start_kernel(void)
 	BUG();
 	pr_warn("7/10");
 	pr_warn("8/10");
+	pr_warn("9/10");
 
 	/* trace_printk can be enabled here */
 	early_trace_init();
-- 
2.17.1

